en/components/categorychart.md

```diff
@@ -0,0 +1,109 @@
+
+However, you can also explicitly specify the chart type by setting the `chartType` to:
+    
+- Line
+- Area
+- Column
+- Point
+- Spline
+- SplineArea
+- StepArea
+- StepLine
+- Waterfall
+
+Another example of the intuitive behavior of the Category Chart control is that you do not need to explicitly set the labels. The Category Chart will use the first appropriate string property that it finds within the data you provided and will use that for the labels.
+
+Built to be easily configurable, it uses the Data Chart control as its engine, which means that it benefits from the high performing and powerful Data Chart. You can also extend upon the category chart to avail of the powerful and extensive features of the data chart control.
+
+<div class="divider"></div>
+
+### Category Chart Demo
+
+<div class="sample-container" style="height: 280px">
+    <iframe id="category-chart-overview-sample-1-iframe" src='{environment:demosBaseUrl}/category-chart-overview-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="category-chart-overview-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">View on StackBlitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+### Dependencies
+
+The category chart is exported as as an `NgModule`, you need to import the _IgxCategoryChartModule_ inside your `AppModule`:
+
+```typescript
+// app.module.ts
+import { IgxCategoryChartModule } from '@infragistics/ignite-ui-xplatform-for-angular/Debug/ES5/ig-categorychart-module';
+
+@NgModule({
+    imports: [
+        ...
+        IgxCategoryChartModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+<div class="divider--half"></div>
+
+### Usage
+
+Now that the category chart module is imported, next step is to bind it to data.
+
+In order to create an igCategoryChart control, you must first have data to bind it to. The following code snippet demonstrates how to create a simple data source. 
+
+```typescript
+var data = [
+ { "CountryName": "China", "Pop1995": 1216, "Pop2005": 1297, "Pop2015": 1361, "Pop2025": 1394 },
+ { "CountryName": "India", "Pop1995": 920, "Pop2005": 1090, "Pop2015": 1251, "Pop2025": 1396 },
+ { "CountryName": "United States", "Pop1995": 266, "Pop2005": 295, "Pop2015": 322, "Pop2025": 351 },
+ { "CountryName": "Indonesia", "Pop1995": 197, "Pop2005": 229, "Pop2015": 256, "Pop2025": 277 },
+ { "CountryName": "Brazil", "Pop1995": 161, "Pop2005": 186, "Pop2015": 204, "Pop2025": 218 }
+];
+
+```
+
+The following code demonstrates how to bind the category chart to the above data. 
+
+```html 
+ <ig-category-chart [dataSource]="data">
```

Comment:
This should be `<igx-category-chart></igx-category-chart>` instead of `<ig-category-chart></ig-category-chart>`